lib/mars.rb

```diff
@@ -0,0 +1,10 @@
+class Mars
+
+attr_reader :size
+
+  def initialize(size)
+    raise ArgumentError.new("Size should be bigger than zero") if size <= 0
+    @size = size
```

Comment:
I think it could be interesting to be more specific about the size. 

I'm assuming it's the diameter, in meters but, I'm may be wrong!